core/src/main/resources/templates/email/messages.properties

```diff
@@ -5,9 +5,16 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 ###############################################################################
-buddy-invitation-subject=Yona buddy invitation from {0}
-buddy-invitation-body.salutation=Hi {0} {1}, become my buddy for Yona!
-buddy-invitation-body.body.p1=As we have talked about, I'd like you to become my buddy in the Yona system. If you didn't install the Yona app yet, please start with doing that. Here are the links:...
-buddy-invitation-body.body.p2=Next, open the below link by tapping on it and selecting Yona as the app to open it with.
-buddy-invitation-body.body.warning=Important: Validate that {0} {1} has really invited you. The mobile number registered with Yona for this user is {2}.
-buddy-invitation-body.body.greeting=Best regards,<br/>{0} {1} | {2}
\ No newline at end of file
+buddy-invitation-subject=Become my friend on Yona!
+buddy-invitation-body.header=Become friends on Yona!
+buddy-invitation-body.p1={0} has invited you to become friends on Yona. Yona enables you to make sure together the Internet remains fun. Follow the steps below and see which goals have been set by {0}. After that, set your own goals.
+buddy-invitation-body.step1=Go to the <a href="{0}" style="color: #2678bf; text-decoration: none;">Apple App Store</a> or the <a href="{1}" style="color: #2678bf; text-decoration: none;">Google Play Store</a> and download Yona.
+buddy-invitation-body.step2=Open the app and &ldquo;join&rdquo;.
+buddy-invitation-body.step3=Return to this mail and click <a href="{0}" style="color: #2678bf; text-decoration: none;">this link</a> to become a friend.
+buddy-invitation-body.warning=<strong>Important</strong>: Verify if the invitation is really from {0} {1} and check the mobile number: <a href="tel:{2}" style="color: #2678bf; text-decoration: none;">{2}</a>.
+buddy-invitation-body.footerHeader=YONA | Challenge your online life.
+buddy-invitation-body.footerText=How long do you want to be online at maximum? Set your own challenges per app or type of activity. How much time do you want to spend on Instagram or WhatsApp? How much time do you want to spend on gaming or shopping? On what do you not want to spend time at all?
```

Comment:
Propose to change
``How long do you want to be online at maximum? Set your own challenges per app or type of activity. How much time do you want to spend on Instagram or WhatsApp? How much time do you want to spend on gaming or shopping? On what do you not want to spend time at all?``
into
``How long do you want to be online at most? Set your own challenges per app or per activity type. How much time do you want to spend on Instagram or WhatsApp? How much time do you want to spend on gaming or shopping? On what do you want to spend no time at all?``